Event/SymfonyEvent.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Swarrot\SwarrotBundle\Event;
+
+use Symfony\Contracts\EventDispatcher\Event;
+use Symfony\Component\EventDispatcher\Event as DeprecatedEvent;
+
+if (interface_exists(Event::class)) {
```

Comment:
this seems wrong to me. `interface_exists` will return true only for an interface, but you extend the class below.

Reply:
@pvgnd could you take a look please?

Reply:
Fixed https://github.com/swarrot/SwarrotBundle/pull/163